src/client/app/home/home.component.html

```diff
@@ -1,6 +1,4 @@
-<p>
-  Howdy! Here's a list of awesome computer scientists. Do you know any others? Add to the list yourself.
-</p>
+<p>Howdy! Here's a list of awesome computer scientists. Do you know any others? Add to the list yourself.</p>
```

Comment:
Could you add some more examples on use (like how do we can translate numbers and variables and/or plurals)